packages/query-store/package.json

```diff
@@ -0,0 +1,37 @@
+{
+	"name": "@evidence-dev/query-store",
+	"version": "1.0.0",
+	"description": "",
+	"main": "dist/index.js",
+	"source": "./src/index.ts",
+	"types": "dist/index.d.ts",
+	"type": "module",
+	"scripts": {
+		"build": "tsc -p ./tsconfig.json",
+		"watch": "tsc -p ./tsconfig.json --watch",
+		"test": "vitest --run",
+		"test:watch": "vitest"
+	},
+	"keywords": [],
+	"author": "The Evidence Engineering Team",
+	"license": "MIT",
+	"devDependencies": {
+		"typescript": "^5.2.2",
+		"vitest": "^0.34.1"
+	},
+	"dependencies": {
+		"@evidence-dev/universal-sql": "workspace:",
+		"@sqltools/formatter": "^1.2.5",
```

Comment:
`nanoid` probably needs to get ripped out;
This package is included because it is used when displaying the queries on the page; there is an issue with it though when trying to use it in an evidence project; so I need to hammer that down at some point.